tests/contract-playground/src/UninitializedLocalVariables.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MITc
```

Comment:
Small typo that I will fix here @TilakMaddy

Reply:
@alexroan Ahh this was `zed` editor's bug . It appended `c` whenever I used a keybinding to navigate away to another file. Thanks for correcting though.